docs/mkdocs/docs/home/exceptions.md

```diff
@@ -295,13 +295,28 @@ When parsing CBOR or MessagePack, the byte vector ends before the complete value
 
 ### json.exception.parse_error.112
 
-Not all types of CBOR or MessagePack are supported. This exception occurs if an unsupported byte was read.
+An unexpected byte was read in a [binary format](../features/binary_formats/index.md) or length information is invalid ([BSON](../features/binary_formats/bson.md)).
 
 !!! failure "Example message"
```

Comment:
"messages"

Reply:
Done in aacd97a1c0190657db231707a2e2be2ac5f02f57.